share/revng/test/configuration/revng-c/segregate-stack-accesses.yml

```diff
@@ -23,6 +23,7 @@ commands:
 
       revng analyze --resume "$OUTPUT" detect-stack-size "$INPUT" -o /dev/null;
 
-      revng artifact --resume "$OUTPUT" make-segment-ref "$INPUT" | FileCheck ${SOURCE}.filecheck.ll;
+      revng artifact --resume "$OUTPUT" make-segment-ref "$INPUT" |
+        zstdcat | llvm-dis | FileCheck ${SOURCE}.filecheck.ll;
```

Comment:
As a convention, we prefer `opt -S` to `llvm-dis`.

Reply:
Done